adrs/2023-12-01.ot-application.md

```diff
@@ -0,0 +1,44 @@
+# Title: Open Truss application
+
+**Status**: Research in progress
+
+**Date**: 2023-12-01
+
+# Context
+
+Open Truss is designed to be powerful framework for building internal tools. Thus far our discussion has been focused on the front-end. After spending some time building the rendering engine, designing the unified query engine, and discussing workflow implementation, it became clear Open Truss will need a significant server-side component to make the system work. In service of enabling this, this ADR explores how we can design Open Truss to support these features.
```

Comment:
This context assumes a level of understanding that the reader might not have. Here are some suggestions

- Link to the Open Truss project
- Explain _how_ or _why_ it is a powerful framework for building internal tools. For instance, what "internal tools" or operational needs does it address that other frameworks do not? 
- Can you list the core functionality of the framework?
- Why has the team decided to use it?